tests/e2e/utils/workspace/ITestWorkspaceUtil.ts

```diff
@@ -11,9 +11,10 @@
 import { WorkspaceStatus } from './WorkspaceStatus';
 
 export interface ITestWorkspaceUtil {
+    cleanUpAllWorkspaces() : void;
     waitWorkspaceStatus(namespace: string, workspaceName: string, expectedWorkspaceStatus: WorkspaceStatus) : void;
     waitPluginAdding(namespace: string, workspaceName: string, pluginId: string) : void;
     removeWorkspaceById(id: string) : void;
     stopWorkspaceById(id: string) : void;
-    getIdOfRunningWorkspace(namespace: string): Promise<string>;
```

Comment:
On my opinion, we should keep the "getIdOfRunningWorkspace" method. In this case we keep possibility of the obtaining single workspace id, that may be helpful when we will extend the functionality of utils. Just keep this method and add your created method.